src/app/shared/components/admin-carriages/carriage-form/carriage-form.component.html

```diff
@@ -0,0 +1,72 @@
+<mat-card>
+  <mat-card-title>
+    <ng-container *ngIf="!carriage">Create Carriage</ng-container>
```

Comment:
Maybe it will be better use ngIf and ngElse, then ngIf and ngIf

Reply:
done